kahuna/public/stylesheets/main.css

```diff
@@ -1265,16 +1265,39 @@ FIXME: what to do with touch devices
     border-left: 1px solid #565656;
     background-color: #444;
 }
+
+.image-details--crop {
+    float: left;
```

Comment:
It'd be good to try and avoid floats as they always seem to create layout issues...